policy/policy-authority-data.go

```diff
@@ -169,9 +169,7 @@ func (padb *PolicyAuthorityDatabaseImpl) allowedByWhitelist(host string) bool {
 
 // CheckHostLists will query the database for white/blacklist rules that match host,
 // if both whitelist and blacklist rules are found the blacklist will always win
-// Returns errNotWhitelisted, errBlacklisted, or errDBFailure for the
```

Comment:
Sorry I missed this the last time around: Looks like you deleted this comment by accident.